project/src/views/new-evaluations/newEvaluationsViews.js

```diff
@@ -24,356 +24,98 @@ const CandidateDetailsForm = function() {
     `;
 }
 
-const TechnicalLevelPicker = function() {
+const TechnicalLevelHeader = function(options){
+    const headings = options.headings.map((el) => `<th>${el}</th>`);
+    return `
+    <tr>
+       ${headings.join('')}
+    </tr>
+    `;
+}
+
+const TechnicalLevelPickerHeaders = function(options = {}){
+    const str = options.map((el) => `<th>${el}</th>`);
```

Comment:
you can do directly return options.map( ....) in here and remove the const str part

Reply:
👍